src/ToData.purs

```diff
@@ -200,6 +201,9 @@ instance ToData a => ToData (Ratio a) where
 instance ToData ByteArray where
   toData = Bytes
 
+instance ToData CborBytes where
```

Comment:
> but it's probably better to use wrap instead

I thought it would be clearer to use the constructor in cases like this one:

```
instance FromData BaseAddress where
  fromData (Bytes res) = baseAddressFromBytes (CborBytes res)
  fromData _ = Nothing
```

Because it makes it obvious to any reader that the bytes are hereby declared to contain cbor. `wrap` doesn't say anything. I am happy to change it to `wrap` though if you prefer it that way.

Reply:
>>>
@adamczykm I see, I was under the impression that CSL types indicated CBOR or raw bytes by the name of the method (for example to_bytes vs bytes). 
>>>

And that seems to be assumed in `Hash.js`/`Hash.purs` too, as there are two distinct functions.

```
exports.scriptHashAsBytes = h => h;
exports.scriptHashToBytes = hashToBytes;
```

So are they the same after all?